### DISCUSSION TITLE
"Only for" button with the same "text-input" capabilities as the "Site list" tab?

### DISCUSSION THREAD

**Discussion by bsmelo** [darkreader/darkreader#7087] [Ideas]
I have seen discussion https://github.com/darkreader/darkreader/discussions/6965 and issues https://github.com/darkreader/darkreader/issues/147 and https://github.com/darkreader/darkreader/issues/373, but I think those (mainly the issues and not the discussion) deal with a larger scope than what I first thought here. Anyway, since I didn't actually read the whole history of those issues, I thought it was better to start a discussion instead of opening a new issue, at least for now.

Is there a reason (is it too complicated in the current state of things, for instance) for the button "Only for" not having the same capabilities as the entries in the "Site list" tab?

The "Site list" tab is very flexible, as I can currently say I don't want the extension running for `project.company.com/wiki` but I do want it running on `project.company.com/jira`. Nevertheless, it would be great if I could actually use `Dark` mode for `project.company.com/wiki` and `Light` mode for `project.company.com/jira`. This, however, is currently not possible, since the "Only for" button is fixed for the whole `project.company.com` part of the URL.

**Comment by Gusted** [comment]
> Is there a reason (is it too complicated in the current state of things, for instance) for the button "Only for" not having the same capabilities as the entries in the "Site list" tab?

Nobody ever asked or proposed such feature - better it has never gone around my mind for that. But given a fast look at it - it will be possible to do it, UI-Wise will be a bit tricky.



> The "Site list" tab is very flexible, as I can currently say I don't want the extension running for `project.company.com/wiki` but I do want it running on `project.company.com/jira`. Nevertheless, it would be great if I could actually use `Dark` mode for `project.company.com/wiki` and `Light` mode for `project.company.com/jira`. This, however, is currently not possible, since the "Only for" button is fixed for the whole `project.company.com` part of the URL.

This also sounds like #2886 which most likely should be closed and re-branded to have some kind of text input or a similar design to Stylus's URL create matcher. 

Could you make a feature request issue for this? So we can properly track it and discuss some details of it.
